src/components/Images.vue

```diff
@@ -104,13 +104,12 @@ export default {
 				return this.getBase64FromImage()
 			}
 
-			if (this.source) {
-				return this.source
-			}
```

Comment:
Ah, we still need to use the source if provided.
If Mail provides an url that is outside dav, we need to use it instead.
Or something like that

```suggestion
			if (this.source && !this.hasPreview) {
				return this.source
			}
```

Reply:
Even better. I incorrectly assumed (from the changes in `PreviewUrl.js`) that we want to use the original file for some reason.